libpod/container_internal_linux.go

```diff
@@ -49,10 +50,15 @@ func (c *Container) mountSHM(shmOptions string) error {
 	return nil
 }
 
-func (c *Container) unmountSHM(mount string) error {
+func (c *Container) unmountExtra(mount string) error {
+	if err := overlay.CleanupMount(mount); err != nil {
```

Comment:
This needs to be separated from SHM unmount as the two cannot use the same field in the database.

Reply:
Still valid?

Reply:
Probably not? We're not directly using this to unmount overlay mounts anymore.

Reply:
Though I'm really confused as to why we retained the `overlay.CleanupMount` bit given we aren't using it for that anymore - should remove this.